### PR TITLE
Add my blog

### PR DESCRIPTION
**Pull request by ShubhamGhanmode**
Pls add my blog.

### CODE REVIEW

**Review by isuruAb — COMMENTED** [review]
please squash your commits and resolve conflicts

File: _data/blogs.yml [hunk 1]
```diff
@@ -292,3 +292,8 @@
   title: Google Code-In: A Platform To Learn
   img: shreyasandurkar.jpg
   author_name: Shreya Sandurkar
+  
+  - url: https://shubham970121859.wordpress.com/
```
Comment (nikit19) [inline _data/blogs.yml]: Please remove the space before ' - '

**Review by nikit19 — CHANGES_REQUESTED** [review]
Make sure that the dimensions of your image is 240 x 240 pixels

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
This branch has conflicts that must be resolved

**Review by anshumanv — CHANGES_REQUESTED** [review]
Resolve conflicts, upload only one rightly sized image.

**Comment by harshit98** [comment]
Closing the PR due to inactivity. Feel free to reopen when you're done with the suggested changes.

**Pull request closed without merging**